### DISCUSSION TITLE
When will Whisky use GPTK2?

### DISCUSSION THREAD

**Discussion by DasJabper** [Whisky-App/Whisky#1024] [General]
since apple introduced GPTK2 only yesterday, I know it will take some time to make it work with that, but when approximately will it work with the new system?

**Comment by SuddenDeathMP** [comment]
Actually you can download it and basically replace GPTK1 with GPTK2, there is even an instruction inside  GPTK2 package

**Whisky:** 
replace Whisky’s copies of the D3DMetal.framework and libd3dshared.dylib found at ~/Library/Application Support/com.isaacmarovitz.Whisky/Libraries/Wine/lib/external/ with the libraries from this distribution:
```
cd ~/Library/Application\ Support/com.isaacmarovitz.Whisky/Libraries/Wine/lib/external
mv D3DMetal.framework D3DMetal.framework-old; mv libd3dshared.dylib libd3dshared.dylib-old
ditto /Volumes/Evaluation\ environment\ for\ Windows\ games\ 2.0/redist/lib/external/ .

```

**Comment by IsaacMarovitz** [comment]
It is already out as of WhiskyWine 2.3.0
